drivers/sensor/Kconfig

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SENSOR
```

Comment:
Please do not use a generic identifier like "SENSOR".  This identifier is specific to the sensor definition in *this* repository and should avoid conflict with identifiers pulled in from future modules that may be added to the Zephyr default manifest.

Reply:
> Please do not use a generic identifier like "SENSOR"

_stet_

https://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_SENSOR.html